scripts/helper/functions.sh

```diff
@@ -62,6 +62,39 @@ get_kafka_cluster_id_from_container()
   return 0
 }
 
+clean_demo_env()
+{
+  local DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+  echo "CLEAN=true -> deleting existing certificates and local Connect Docker image generated by cp-demo"
```

Comment:
Logic OK, but the output can be confusing.  If I start with a squeaky-clean working copy (no certs) and run:

```
$ CLEAN=false scripts/start.sh
CLEAN=true -> deleting existing certificates and local Connect Docker image generated by cp-demo
#    -o xtrace

# Cleanup files
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd
 dirname "${BASH_SOURCE[0]}"
(cd "${DIR}" && rm -f *.crt *.csr *_creds *.jks *.srl *.key *.pem *.der *.p12)
```

So I asked for `CLEAN=false`, which can't happen because already clean, and the output is immediately `CLEAN=true -> deleting existing certificates ...`, which contradicts, and claims to be deleting existing certificates, but there are none.

The same message appears on a clean repo without the `CLEAN=false` - less contradictory, but still claims to be deleting existing certificates, and there are none.

So the logic is OK, but I think we'll need to do more explaining of the logic in the output here.

Reply:
👍 https://github.com/confluentinc/cp-demo/pull/273/commits/48c658b5f4cb1c9f405e7ba006ecd7fa5fe05dee improves UX

Reply:
⭐ 🍰 ⭐